contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/checkstyle_integration_test.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class PythonLintIntegrationTest(PantsRunIntegrationTest):
+
+  TESTPROJECT = "testprojects/src/python/interpreter_selection/python_3_selection_testing{}"
+
+  def test_lint_runs_for_py2_only(self):
+    command = ['lint', self.TESTPROJECT.format(':main_py2')]
+    pants_run = self.run_pants(command=command)
```

Comment:
Oof. Integration tests are easy to setup but we really do pay for these in CI time. How about building on the existing unit test?

Reply:
I'll do that.